### ISSUE TITLE
panic in `build-manifest`

### ISSUE THREAD

**Issue by rosenhouse** [cloudfoundry/bosh-cli#46]
```
bosh-cli build-manifest --var-errs --var-file=/tmp/creds.yml cf-deployment/cf-deployment.yml
```
gives
```
[CLI] 2016/11/13 18:02:45 ERROR - Panic: runtime error: slice bounds out of range
********************
goroutine 1 [running]:
runtime/debug.Stack(0x706d00, 0xc4203a78f0, 0x28)
	/usr/local/Cellar/go/1.7.3/libexec/src/runtime/debug/stack.go:24 +0x79
main.handlePanic()
	/Users/pivotal/go/src/github.com/cloudfoundry/bosh-cli/main.go:110 +0x2e1
panic(0x687c60, 0xc420010150)
	/usr/local/Cellar/go/1.7.3/libexec/src/runtime/panic.go:458 +0x243
github.com/cloudfoundry/bosh-cli/cmd.Cmd.Execute.func1(0xc42014ea40)
	/Users/pivotal/go/src/github.com/cloudfoundry/bosh-cli/cmd/cmd.go:40 +0xe7
panic(0x687c60, 0xc420010150)
	/usr/local/Cellar/go/1.7.3/libexec/src/runtime/panic.go:458 +0x243
github.com/cloudfoundry/bosh-cli/vendor/github.com/cppforlife/go-patch/patch.FindOp.Apply(0x0, 0x0, 0x0, 0x681340, 0xc420227e00, 0x681340, 0xc420227e00, 0x0, 0x0)
	/Users/pivotal/go/src/github.com/cloudfoundry/bosh-cli/vendor/github.com/cppforlife/go-patch/patch/find_op.go:20 +0x17ce
github.com/cloudfoundry/bosh-cli/vendor/github.com/cppforlife/go-patch/patch.(*FindOp).Apply(0xc4202ab520, 0x681340, 0xc420227e00, 0x681340, 0xc420227e00, 0x0, 0x0)
	<autogenerated>:5 +0x78
github.com/cloudfoundry/bosh-cli/vendor/github.com/cppforlife/go-patch/patch.Ops.Apply(0xc4202ab4e0, 0x2, 0x2, 0x681340, 0xc420227e00, 0x0, 0x0, 0x0, 0xc41ffeaa56)
	/Users/pivotal/go/src/github.com/cloudfoundry/bosh-cli/vendor/github.com/cppforlife/go-patch/patch/ops.go:20 +0x65
github.com/cloudfoundry/bosh-cli/director/template.Template.Evaluate(0xc4202c0000, 0x95d0, 0xfe00, 0xc4202271d0, 0xc4202ab4e0, 0x2, 0x2, 0x101, 0x0, 0xc4202271d0, ...)
	/Users/pivotal/go/src/github.com/cloudfoundry/bosh-cli/director/template/template.go:39 +0x101
github.com/cloudfoundry/bosh-cli/cmd.BuildManifestCmd.Run(0xaa5520, 0xc42019a030, 0xaa6b60, 0xc42019c040, 0xc4202c0000, 0x95d0, 0xfe00, 0x0, 0x0, 0x0, ...)
	/Users/pivotal/go/src/github.com/cloudfoundry/bosh-cli/cmd/build_manifest.go:32 +0x26e
github.com/cloudfoundry/bosh-cli/cmd.Cmd.Execute(0x7c3090, 0x6978ef, 0xe, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/pivotal/go/src/github.com/cloudfoundry/bosh-cli/cmd/cmd.go:202 +0x1e79
main.main()
	/Users/pivotal/go/src/github.com/cloudfoundry/bosh-cli/main.go:35 +0x6cc

********************
Exit code 1
```

To reproduce, just follow the [usage instructions for cf-filler](https://github.com/rosenhouse/cf-filler).

**Comment by rosenhouse** [comment]
just did a `git bisect`, got
```
4f4e3075232c7e46191fbcd0a786bba83e3e4476 is the first bad commit
commit 4f4e3075232c7e46191fbcd0a786bba83e3e4476
Author: dmitriy kalinin <dmitriy@pivotallabs.com>
Date:   Thu Nov 10 08:31:51 2016 -0800

    add --out to build-manifest to pull out specific values

:040000 040000 453464ae4590707f0c346b1ef370dc441d1a4073 1f551f57c9fe60df42809d86d566de669c6181ce M	cmd
:100644 100644 fa99760873a1fb0185c61a053ef50b5770eb3a55 f024dbc7b8292b62a0aae1641253f01f0e789eb9 M	deps.txt
:040000 040000 b286111419ffa7251fe67e4b9b4e91ab0e39b62b 675a7bf33fa82319081e676fee8659f3798ee450 M	director
:040000 040000 6d8ca5153d8c065e55e6dde35f44399425dbb36a ab638177f6e728613d880db3d6988815a3601d56 M	vendor
```

**Comment by cppforlife** [comment]
Fixed in v0.0.107.
